scripts/build_scripts/create_venv.sh

```diff
@@ -21,3 +21,9 @@ python3 -m pip install -r $(pwd)/tt_metal/python_env/requirements-dev.txt
 echo "Installing tt-metal"
 pip install -e .
 pip install -e ttnn
+
+echo "Generating git hooks"
+pre-commit install
+pre-commit install --hook-type commit-msg
+
+echo "If you want stubs, run ./scripts/build_scripts/create_stubs.sh"
```

Comment:
ooo interesting idea, should point out that this can only be done after C++ build is done

Reply:
there's a line in `create_stubs.sh`

`echo "!! Did you build metal and create python_env? !!"`

Reply:
I can check if pybinds are installed. Checking if they made the python_env seems like too much handholding?

Reply:
Maybe the check could be trying `python3 -c 'import ttnn._ttnn` or something like that, and something similar for tt_lib? Will leave up to you

And agree on too much handholding on python_env. Note that @TT-billteng and I are trying to say python env creation should be independent of pybinds/stubs install itself

Reply:
now checking for pybinds before installing stubs. 

Is end goal to just have `create_env.sh` create python_env, and then devs can run `pip install -e ...` themselves?

Reply:
I think `create_env.sh` can still `pip install ...` because it assumes a certain way of creating the env anyway.

I think the way to think about `create_env.sh` is a convenient way to creating a local dev environment that works and is familiar. Also, a way to create consistent envs for CI (until of course if we decide to use something different like conda)

If people want to use something like conda for local dev, then that's up to them to figure out how to do.